test/tests/ver-1b/comparison_ver-1b.py

```diff
@@ -11,7 +11,7 @@
 gs = gridspec.GridSpec(1,1)
 ax = fig.add_subplot(gs[0])
 
-tmap_sol = pd.read_csv("./gold/u_vs_t.csv")
+tmap_sol = pd.read_csv("ver-1b_csv.csv")
```

Comment:
```suggestion
tmap_sol = pd.read_csv("./gold/ver-1b_csv.csv")
```
We settled on using the gold files.